packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts

```diff
@@ -768,21 +783,6 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
             this.state.mouseDisposer = undefined;
         }
     }
-
-    private trySelectSingleImage(selection: RangeSelection) {
```

Comment:
When selecting the image with Ctrl + Arrow Keys now we do not show image selection and instead we show range selection. Is this expected? 

I believe it is different than Word, in Word as soon as the image is selected with either Shift + Arrow, Word adds the wrapper for resizing the image.